glue/flagging_script_glue/flagging.py

```diff
@@ -270,26 +266,6 @@ def pricing_info(
     return df
 
 
-def special_flag(df: pd.DataFrame) -> pd.DataFrame:
-    """
-    Creates column that checks whether there is a special flag for this record.
-    Inputs:
-        df (pd.DataFrame): dataframe to add flags onto
-    Outputs:
-        df (pd.DataFrame): dataframe with 'special_flags' column
-    """
-    cond = [
-        (df["sv_name_match"] != "No match"),
-        (df["sv_short_owner"] == "Short-term owner"),
-        (df["sv_transaction_type"] == "legal_entity-legal_entity"),
-    ]
-    labels = ["Family sale", "Home flip sale", "Non-person sale"]
-
-    df["sv_special_flags"] = np.select(cond, labels, default="Not special")
-
-    return df
-
-
```

Comment:
As far as I could tell, this function was serving no purpose in the pipeline, and I couldn't find any references to the `sv_special_flags` column in the repo.